### PR TITLE
[TS] LPS-77040

### PR DESCRIPTION
**Pull request by brandizzi**

### CODE REVIEW

**Comment by jeyvison** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jeyvison** [comment]
ci:test

File: modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/scheduler_event_recorder.js [hunk 1]
```diff
@@ -78,6 +79,13 @@ AUI.add(
 
 				NAME: 'scheduler-event-recorder',
 
+				ATTRS: {
+					dateFormat: {
```
Comment (jeyvison) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/scheduler_event_recorder.js]: @ling-alan-huang Hey Ling, Can you please move this attribute to the already created ATTRS block? Check [this line.](https://github.com/brandizzi/liferay-portal/blob/f4768e3379c1dc783577bbd7d071df7b81e787c0/modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/scheduler_event_recorder.js#L26)

cc/ @hhuijser

**Comment by ling-alan-huang** [comment]
Hi @jeyvison ,
Thank you for your catching this, I will resend to you.
Please see https://github.com/jeyvison/liferay-portal/pull/24
cc/ @yuhai, @hhuijser, @brandizzi

**Pull request closed without merging**